docs/criteria-for-first-major-release.md

```diff
@@ -0,0 +1,35 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Criteria for a First Major Release
```

Comment:
`First Major Release` should be replaced with `Promoted Release` as per [this decision](https://wiki.hyperledger.org/pages/viewpage.action?pageId=24779174)